src/test/java/seedu/address/testutil/TypicalPersons.java

```diff
@@ -19,6 +29,9 @@
 import seedu.address.model.person.ReadOnlyPerson;
 import seedu.address.model.person.exceptions.DuplicatePersonException;
 
+
+
```

Comment:
Why did you add these spacings? Why did you add on to this file? (see above for the same comment)